frontend/src/App.jsx

```diff
@@ -16,8 +16,11 @@ function App() {
         <Route path="/profile" element={<Profile />} />
         <Route path="/profile/:profileId" element={<Profile />} />
         <Route path="/register" element={<Register />} />
-        <Route path="/register/artist" element={<SignupArtist />} />  
-        <Route path="/register/artist/validation" element={<RegisterArtist />} />
+        <Route path="/register/artist" element={<SignupArtist />} />
```

Comment:
Why do I feel like I have a file in a PR, you haven't left the dev?

Reply:
That's because they didn't: I've read the same three lines of code for 3 PRs in the last five minutes